src/PantherTestCaseTrait.php

```diff
@@ -140,11 +142,23 @@ public static function startWebServer(array $options = []): void
             'webServerDir' => self::getWebServerDir($options),
             'hostname' => $options['hostname'] ?? self::$defaultOptions['hostname'],
             'port' => (int) ($options['port'] ?? $_SERVER['PANTHER_WEB_SERVER_PORT'] ?? self::$defaultOptions['port']),
-            'router' => $options['router'] ?? $_SERVER['PANTHER_WEB_SERVER_ROUTER'] ?? self::$defaultOptions['router'],
+            'params' => [
+                'router' => $options['router'] ?? $_SERVER['PANTHER_WEB_SERVER_ROUTER'] ?? self::$defaultOptions['router'],
```

Comment:
since router is only needed for the PHP Webserver, I thought it'd be better to introduce a webserver `params` variable here

Reply:
reverted and added --passthru to symfony-cli webserver